Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/PingServiceImpl.java

```diff
@@ -79,7 +79,7 @@ public boolean ping(String url) {
             con.disconnect();
             return true;
         } catch (Exception ex) {
-            LOG.warn("Problem pinging endpoint: " + ex.getLocalizedMessage(), ex);
+            LOG.warn("Problem pinging endpoint: {}", ex.getLocalizedMessage(), ex);
```

Comment:
Catching this exception doesn't necessarily indicate an error; it could be that the service we're trying to reach isn't deployed / available, and in that case the exception is expected behavior.